scripts/deploy/common.rb

```diff
@@ -31,3 +31,52 @@ def open_url(url)
     puts "Opening url #{url}"
     `open #{url}`
 end
+
+def delete_git_branch(branch_name)
+    # Ensure we are not on the same branch that we're trying to delete
+    execute("git checkout #{@branch}")
+
+    # Actually delete the branch
+    execute("git push origin --delete #{branch_name}")
+    execute("git branch -D #{branch_name}")
+end
+
+def switch_to_new_branch(branch_name)
+    # Ensure a different version of this branch doesn't already exist.
+    delete_git_branch(branch_name)
+
+    execute_or_fail("git checkout -b #{branch_name}")
+end
+
+def create_pr(
+    pr_branch,
+    pr_title,
+    pr_description,
+    user_message
+)
+    response = github_login.create_pull_request(
+        "stripe/stripe-android",
+        "master",
+        pr_branch,
+        pr_title,
+        pr_description,
+        draft: @is_dry_run,
+    )
+
+    pr_url = response.html_url
+    open_url(pr_url)
+
+    rputs user_message
+    wait_for_user
+end
+
+private def github_login
+  token = `fetch-password -q bindings/gh-tokens/$USER`
+  if $?.exitstatus != 0
+    puts "Couldn't fetch GitHub token. Follow the Android SDK Deploy Guide (https://go/android-sdk-deploy) to set up a token. \a".red
```

Comment:
What's the \a?

Reply:
apparently it is the bell character! https://en.wikipedia.org/wiki/Bell_character

So apparently it'll make a sound or visual alert in your terminal. This is inconsistent with what we do everywhere else though and it's just copied over.

I'm going to delete this for now and then add better error messages for missing passwords when I add the permissions check step.

Reply:
TIL

Reply:
removed this for consistency